### PR TITLE
feat: Add Desktop generated code for the Welcome page

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
## PR Type

What kind of change does this PR introduce?

- Feature

## Description

Add Desktop generated code for the Welcome page

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/ToDo.UI/Views/WelcomePage.xaml [hunk 1]
```diff
@@ -1,28 +1,116 @@
...
 
-	<utu:AutoLayout Padding="10,10,10,10">
-		<utu:AutoLayout Padding="20,60,20,60" Orientation="Horizontal" utu:AutoLayout.PrimaryAlignment="Stretch">
-			<Image Source="ms-appx:///Assets/Welcome/welcome_uno.png" Stretch="Uniform" utu:AutoLayout.CounterAlignment="Start" utu:AutoLayout.PrimaryAlignment="Stretch" utu:AutoLayout.CounterLength="310" />
+	<Grid>
+		<VisualStateManager.VisualStateGroups>
+			<VisualStateGroup>
+				<VisualState x:Name="Wide">
+					<VisualState.StateTriggers>
+						<AdaptiveTrigger MinWindowWidth="{StaticResource WideMinWindowWidthh}" />
```
Comment (Xiaoy312) [inline src/ToDo.UI/Views/WelcomePage.xaml:19]: "WideMinWindowWidt**hh**" having a stroke?

Comment (Xiaoy312) [reply src/ToDo.UI/Views/WelcomePage.xaml:19]: also where is the resource defined? it doesnt appear to be in uno or uwp

**Review by carldebilly — APPROVED** [review]

File: src/ToDo.UI/Views/WelcomePage.xaml [hunk 2]
```diff
@@ -1,28 +1,116 @@
...
+					   utu:AutoLayout.CounterAlignment="Start"
+					   utu:AutoLayout.PrimaryAlignment="Stretch"
+					   utu:AutoLayout.CounterLength="310" />
+			</utu:AutoLayout>
+			<utu:AutoLayout Spacing="20"
+							Padding="0,40,0,40">
+				<TextBlock Foreground="{ThemeResource MaterialOnSecondaryMediumBrush}"
+						   TextAlignment="Center"
+						   TextWrapping="Wrap"
+						   Text="Welcome to&#xA;Uno To Do!"
```
Comment (Xiaoy312) [inline src/ToDo.UI/Views/WelcomePage.xaml:60]: not sure if it is a good idea to keep newline literal in them, with text wrap
shouldnt we just pick one not both?

**Review by Xiaoy312 — COMMENTED** [review]
normally, having 2 two that switches visibility for "adaptiveness" is not great, since it would be quite jarring
but i guess it is fine in this case, since we are codegen-ing both parts

File: src/ToDo.UI/Views/WelcomePage.xaml (hunk 1 above)
Comment (agneszitte-nventive) [reply src/ToDo.UI/Views/WelcomePage.xaml:19]: @Xiaoy312 defined here, I will adjust the typo 
https://github.com/unoplatform/uno.todo/blob/aedcb706541602fd5648c599e7680aa79e238415/src/ToDo.UI/App.xaml#L39

**Comment by agneszitte-nventive** [comment]
> normally, having 2 two that switches visibility for "adaptiveness" is not great, since it would be quite jarring but i guess it is fine in this case, since we are codegen-ing both parts

Yeah we can update it later on if we manage to find a better solution. For now it's the easiest to be updated in case it's needed

File: src/ToDo.UI/Views/WelcomePage.xaml (hunk 2 above)
Comment (agneszitte-nventive) [reply src/ToDo.UI/Views/WelcomePage.xaml:60]: @carldebilly what do you think ? regarding the figma plugin part for the generated code

**Pull request merged**